bucket/todotxt-net.json

```diff
@@ -3,7 +3,7 @@
     "description": "Implementation of todo.txt for Windows using the .NET framework",
     "version": "3.3.1",
     "license": "BSD-2-Clause-FreeBSD",
-    "url": "https://github.com/benrhughes/todotxt.net/releases/download/v3.3.1/todotxt-portable.zip",
+    "url": "https://github.com/benrhughes/todotxt.net/releases/download/v3.3.1/todotxt-portable-3.3.1.zip",
     "hash": "1c0fe0d37d5519af0e701aebcd47ee17a39751386b18f8e216976c8f75bcf636",
```

Comment:
Hash is also different.

```suggestion
    "hash": "1b85b951595ec1416b5a63553685f3d2a6a25dfea34128c58e93ee184729187c",
```

![A](https://i.imgur.com/pUVTS7M.png)

Reply:
Thanks! Forgot to check the hash, sorry.